node/src/bin/manager.rs

```diff
@@ -1585,3 +1560,222 @@ async fn main() -> anyhow::Result<()> {
 fn parse_duration_in_secs(s: &str) -> Result<Duration, ParseIntError> {
     Ok(Duration::from_secs(s.parse()?))
 }
+
+// A temporary conversion until all the commands are migrated to the core crate.
+fn make_graphman_context(ctx: Context) -> GraphmanContext {
+    let logger = ctx.logger.clone();
+    let notification_sender = ctx.notification_sender();
+    let (store, pool) = ctx.store_and_primary();
+
+    GraphmanContext {
+        pool,
+        notification_sender,
+        store,
+        logger,
+    }
+}
+
+async fn execute_deployment_info(
+    ctx: Context,
+    deployment: Option<DeploymentSearch>,
+    current: bool,
+    pending: bool,
+    status: bool,
+    used: bool,
+    all: bool,
+) -> anyhow::Result<()> {
+    use graphman::deployment_search::DeploymentFilters;
+    use graphman::deployment_search::DeploymentSelector;
+
+    let deployment = match deployment {
+        Some(deployment) => make_deployment_selector(deployment),
+        None if all => DeploymentSelector::All,
+        None => bail!("Please specify a deployment or use --all to list all deployments"),
+    };
+
+    let filters = DeploymentFilters {
+        included_versions: Some(make_deployment_version_filter(current, pending, used)),
+    };
+
+    let status = if !status {
+        DeploymentStatusSelector::Skip
+    } else {
+        DeploymentStatusSelector::Load
+    };
+
+    let command = DeploymentInfoCommand {
+        deployment,
+        filters,
+        status,
+    };
+
+    let ctx = make_graphman_context(ctx);
+    let deployments = command.execute(ctx).await?;
+
+    print_deployments(deployments);
+
+    Ok(())
+}
```

Comment:
IMHO, it would be better to stick with the current convention of keeping the implementation of CLI commands in their own files instead of putting them into `manager.rs`

Reply:
Fixed